projects/bloaty/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER jhaberman@gmail.com
+RUN apt-get update && apt-get install -y cmake ninja-build g++
```

Comment:
why is g++ needed?

Reply:
Before I added that, C++ headers were missing (I got compile failures for missing includes like `<algorithm>`). I wasn't sure the right package to install to get C++ headers, but I noticed a couple other projects installed `g++`, and that seemed to do the trick.